### ISSUE TITLE
glob renders at most one @font-face, the last one.

### ISSUE THREAD

**Issue by kolodziejczakM** [cristianbote/goober#102]
As you can see in the screenshot only last @font-face has been put into <style id="_goober">:
 
![@font_face_in_glob](https://user-images.githubusercontent.com/16230707/73599901-9b615d80-4549-11ea-979b-24a2ff30fee6.png)

Reproduced here: https://codesandbox.io/s/interesting-rubin-twrr1

**Comment by cristianbote** [comment]
Hey @kolodziejczakM, thank you for opening this issue. Undoubtedly this is an issue. ☹️  and It makes total sense for it to happen. Internally, `goober`, tokenises each selector or definition into an object key which get's serialised into CSS.

This is definitely something that `goober` should handle seamlessly. Until there's a fix, I think you could instead of just one `glob` call to call it sequentially with each `font-face` definition, maybe? I'll give it a try and come back with results.

**Comment by kolodziejczakM** [comment]
I've managed to work around this problem by adding fonts separately without even using goober:
 ```
const s = document.createElement('style');
 s.id = '_fonts';
 s.appendChild(document.createTextNode(`${fontFaces}`));
 document.head.appendChild(s);
```
Nonetheless I decided to raise an issue so you @cristianbote know about its existence.
I'm not going to PR it though, at least not in the near future.

**Comment by cristianbote** [comment]
Based on #106 and #109 now each `glob` call it's tokenized and appended if it's fresh. So, that means the sequential call to `glob` now works. Fixed in `goober@1.7.0` 🎉 

Checkout this sandbox https://codesandbox.io/s/billowing-bush-zf3w5
